public/views/play-with.html

```diff
@@ -0,0 +1,53 @@
+<nav id="mainNav" class="navbar navbar-default navbar-fixed-top navbar-custom">
+  <div class="container custom-container">
+    <!-- Brand and toggle get grouped for better mobile display -->
+    <div class="navbar-header page-scroll">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+      <span class="sr-only">Toggle navigation</span>
+      Menu
+      <i class="fa fa-bars"></i>
+      </button>
+      <a class="navbar-brand-custom" href="#">
+      <img alt="Brand" src="./img/cards-for-humanity.png" class="img-responsive" width="70">
+      </a>
+    </div>
+    <!-- Collect the nav links, forms, and other content for toggling -->
+    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1" landing>
+      <ul class="nav navbar-nav navbar-right">
+        <li class="hidden">
+          <a href="#page-top"></a>
+        </li>
+        <li class="page-scroll">
+          <a href="/">HOME</a>
+        </li>
+        <li class="page-scroll">
+          <a href="/#!/#about">ABOUT THE GAME</a>
```

Comment:
Ok, I have tried that but it would not work on the sign in page because the div is not on that page
so I have to redirect to login page using the "/#! and then to the about div "#about"